<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>634</width>
    <height>752</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>AMT&amp;C Helmholz Coils Magnet measurements</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="Layout_1h">
      <item>
       <spacer name="hSpacer_1">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QLabel" name="lbl_MotorPort">
        <property name="minimumSize">
         <size>
          <width>100</width>
          <height>30</height>
         </size>
        </property>
        <property name="text">
         <string>Motor port</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="cBox_MotorPort">
        <property name="minimumSize">
         <size>
          <width>100</width>
          <height>30</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="lbl_SensorPort">
        <property name="minimumSize">
         <size>
          <width>100</width>
          <height>30</height>
         </size>
        </property>
        <property name="text">
         <string>Sensor port</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="cBox_SensorPort">
        <property name="minimumSize">
         <size>
          <width>100</width>
          <height>30</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="hSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="Layout_2v">
      <item>
       <layout class="QHBoxLayout" name="hLayout_2_1">
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="pBtn_Show">
          <property name="minimumSize">
           <size>
            <width>150</width>
            <height>80</height>
           </size>
          </property>
          <property name="text">
           <string>Show Data</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pBtn_Save">
          <property name="minimumSize">
           <size>
            <width>150</width>
            <height>80</height>
           </size>
          </property>
          <property name="text">
           <string>Save Data</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="hLayout_2_2">
        <item>
         <widget class="QPushButton" name="pBtn_GetData">
          <property name="minimumSize">
           <size>
            <width>100</width>
            <height>80</height>
           </size>
          </property>
          <property name="text">
           <string>1. Get data</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="hLayout_2_3">
        <item>
         <widget class="QPushButton" name="pBtn_Outliers">
          <property name="minimumSize">
           <size>
            <width>100</width>
            <height>80</height>
           </size>
          </property>
          <property name="text">
           <string>2.1. Выбросы</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pBtn_Integrate">
          <property name="minimumSize">
           <size>
            <width>100</width>
            <height>80</height>
           </size>
          </property>
          <property name="text">
           <string>2.2. Интеграл</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pBtn_Slope">
          <property name="minimumSize">
           <size>
            <width>100</width>
            <height>80</height>
           </size>
          </property>
          <property name="text">
           <string>2.3. Наклон</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="Layout_3h"/>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>634</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuMenu">
    <property name="title">
     <string>Menu</string>
    </property>
   </widget>
   <addaction name="menuMenu"/>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
